server/database/config/connection.js

```diff
@@ -0,0 +1,23 @@
+const {Pool} = require('pg'); 
+require('env2')('./config.env');
+let dbUrl = ''; 
+switch(process.env.NODE_ENV) {
+    case 'production' :
+        dbUrl = process.env.db_production_url;
```

Comment:
* rename the DATABASE URL to 
     - for production => DATABASE_URL
     - for development => DEV_DB_URL
     - for test => TEST_DB_URL
``` environment constants in the scope of your app.
It's a convention to name constants in capital letters, The underscore just makes it more readable
 ```